docs/build.md

```diff
@@ -0,0 +1,51 @@
+# Building the Kimai Docker
+
+The same docker file is used to build all the tagged images and is configured by combination of build arguments and 
+targets.
+
+## Targets
+
+The docker file has many staging targets but two functional builds are `prod` and `dev` which correspond to the 
+prod and development environments as outlined in the Kimai documentation.  The default is `prod`
+
+    docker build --target=prod .
+    docker build --target=dev .
+
+## Build Arguments
+
+### BASE
+
+  * `BASE=apache-debian`
+  * `BASE=fpm-alpine`
+
+Selects which PHP wrapper to use.  The Apache Debian version bundles an Apache server, and the mod-php wrapper based
+on a debian buster image.  The fpm-alpine version provides the fast CGI version of PHP based on an alpine image.
+
+The Apache/Debian image is bigger (~940mb) but does not require a second container to provide http services.  Use this
+image for development, tests or evaluation.
+
+The FPM image is smaller (~640mb) but requires a web server to provide the http services.  Use this image in production
+and see the [docker-compose](docker-compose.md) page for setting up a http server.   
```

Comment:
The three whitespaces after `"server.   "` violates markdownlint rule [MD009 - Trailing spaces](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md009).